### ISSUE TITLE
Update function alias list in documentation

### ISSUE THREAD

**Issue by Girgias** [php/php-tasks#25]
We have an appendix which lists all the function aliases https://www.php.net/manual/en/aliases.php however it seems to be unmaintained. But I imagine it should be relatively easy to generate this page now that we have this information available in the stubs.

**Comment by kocsismate** [comment]
I agree, it has become easy to generate this page from stubs, as long as we only do this for function-function aliases. 🙂

Unfortunately, there's an issue with function-method aliases that stubs and the manual report aliases differently: while stubs always define the functions as base, the manual usually reports them as aliases, e.g. oci8 on this page, but ext/date or iirc mysqli also does the same.

**Comment by cmb69** [comment]
Regarding the docs: if the methods are documented as aliases of the functions, there is no nice class overview page (e.g. https://www.php.net/xmlwriter). If the functions are documented as aliases of the methods, the search functionality is broken. IMHO, the former is worse, but we may consider to fix/improve the search.
